run-build.sh

```diff
@@ -137,6 +123,15 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+# The first 'pass' call to "dotnet msbuild build.proj" has a hard-coded "WriteDynamicPropsToStaticPropsFiles" target
+#    therefore, this call should not have other targets defined. Remove all targets passed in as 'extra parameters'.
+argsnotargets=( )
+for element in ${args[@]} do
+  if [[ $element != /t:* ]]; then
```

Comment:
I know we only use `/t: but should we defensively check `/target` as well? Also, I think msbuild switches are case insensitive.

Reply:
@nguerrera 
For PS1: if((-not $param.StartsWith("/t")) -and (-not $param.StartsWith("/T"))) 
For SH: if [[ $arg != '/t'* ]] && [[ $arg != '/T'* ]]; then  

That should do it...

Reply:
I see. Feels a little clumsy that it's StartsWith /t. It would match /toolsversion as well. Also, msbuild switches can start with - instead of /.